src/utilities/app-bridge/context.tsx

```diff
@@ -0,0 +1,6 @@
+import {createContext} from 'react';
+import {ClientApplication} from '@shopify/app-bridge';
+
+export const AppBridgeContext = createContext<
+  ClientApplication<{}> | undefined
+>(undefined);
```

Comment:
We set the default for AppBridge to be undefined, but use null for sticky-manager and scroll-lock-manager. Are we able to use a consistent type for both of these optional fields?

Reply:
I'll take a look, I left it as is because we're removing it in `v5` but it might be a simple change.

Reply:
In wonder if instead we could say createContext for stickymanager and scrollockmanager should both always return a default value of an instance of their respective classes, that way we wouldn't have to deal with "what if the sticky manager was null" logic in components that consume the context provider.

But that can be done in a follow up PR